src/vivarium_gates_mncnh/components/intervention.py

```diff
@@ -463,14 +459,9 @@ def get_excess_shift_lookup_table(self, builder: Builder) -> LookupTable:
     def get_relative_risk_source(self, builder: Builder) -> Callable[[pd.Index], pd.Series]:
         return lambda index: pd.Series(1.0, index=index)
 
-    def get_target_modifier(
-        self, builder: Builder
-    ) -> Callable[[pd.Index, pd.Series], pd.Series]:
-        def adjust_target(index: pd.Index, target: pd.Series) -> pd.Series:
-            affected_rates = target + self.effect(index)
-            return affected_rates
-
-        return adjust_target
+    def adjust_target(self, index: pd.Index, target: pd.Series) -> pd.Series:
```

Comment:
VPH now uses adjust_target directly, which is change from when NO's AdditiveRisk model was written.